### PR TITLE
Fixes #232 Add go struct refer to java.util.UUID 

### LINKED ISSUE #232: Add go struct refer to java.util.UUID
**What would you like to be added**:

Add go struct refer to java.util.UUID

**Why is this needed**:

to support more jdk classes

### PR DESCRIPTION
**Pull request by zhangymPerson**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,32 +1,16 @@
 language: go
 
-jobs:
```
Comment (wongoo) [inline .travis.yml:3]: why remove jdk？

File: .travis.yml [hunk 2]
```diff
@@ -1,32 +1,16 @@
...
-  include:
-    - language: java
-      jdk: openjdk8
-
 go:
+  - "1.11"
   - "1.12"
 
-os: linux
```
Comment (wongoo) [inline .travis.yml:11]: why update .travis.yml ?

File: java_util/UUID.go [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+type UUID struct {
+	MostSigBits  int64 `hessian:"mostSigBits"`
+	LeastSigBits int64 `hessian:"leastSigBits"`
+}
+
+func (UUID) JavaClassName() string {
+	return "java.util.UUID"
+}
+
+func (UUID) Error() string {
```
Comment (wongoo) [inline java_util/UUID.go:34]: why impl Error()?

File: java_util/UUID.go [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+// hexDigit               =
+//       "0" | "1" | "2" | "3" | "4" | "5" | "6" | "7" | "8" | "9"
+//       | "a" | "b" | "c" | "d" | "e" | "f"
+//       | "A" | "B" | "C" | "D" | "E" | "F"
+//
+//Returns:
+//A string representation of this UUID
+func (uuid UUID) ToString() string {
+	uuidStr := fmt.Sprintf("%v-%v-%v-%v-%v",
+		digits((uuid.MostSigBits>>32), 8),
```
Comment (wongoo) [inline java_util/UUID.go:61]: should remove more brackets

File: java_util_test.go [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+ */
+
+package hessian
+
+import (
+	"testing"
+)
+
+func TestJavaUtil(t *testing.T) {
+	//testDecodeFramework(t, "javautilUUID", &java_util.UUID{LeastSigBits: int64(2020), MostSigBits: int64(8067822391307271002)})
```
Comment (wongoo) [inline java_util_test.go:25]: why comment？

**Comment by wongoo** [comment]
ci error: `java_util/UUID.go:68:20: invalid operation: int64(1) << (digits * 4) (shift count type int, must be unsigned integer)`

**Comment by wongoo** [comment]
@zhangymPerson can u add another unit test , which check whether the uuid string generated by java is equal to that generated by golang ?

File: java_util/UUID.go [hunk 6]
```diff
@@ -0,0 +1,71 @@
+/*
```
Comment (AlexStocks) [inline java_util/UUID.go:1]: pls  rename this file to uuid.go

File: java_util/UUID_test.go [hunk 7]
```diff
@@ -0,0 +1,49 @@
+/*
```
Comment (AlexStocks) [inline java_util/UUID_test.go:1]: pls  rename this file to uuid_test.go

File: java_util_test.go [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package hessian
+
+import (
```
Comment (AlexStocks) [inline java_util_test.go:20]: pls split this import block.

**Comment by wongoo** [comment]
> @zhangymPerson can u add another unit test , which check whether the uuid string generated by java is equal to that generated by golang ?

@zhangymPerson  maybe u do not got what I means, u response uuid string like the following and check whether it's equal to `&java_util.UUID{LeastSigBits: int64(-7160773830801198154), MostSigBits: int64(459021424248441700)}.ToString()`

```java
    public static String javautilUUIDString() {
        UUID uuid = new UUID(459021424248441700L, -7160773830801198154L);
        return uuid.toString();
    }
```

**Review by wongoo — APPROVED** [review]

**Review by AlexStocks — APPROVED** [review]

**Review by gaoxinge — APPROVED** [review]

**Pull request merged**